project/gh-action/python/app.py

```
@@ -0,0 +1,23 @@
+    return sum(float(item.get('price', 0)) for item in items)
+
+def validate_user_input(user_data):
+    """Safely validate user input with proper checks."""
+    required_fields = ['username', 'email']
+    
+    if not all(field in user_data for field in required_fields):
+        raise ValueError("Missing required fields")
+    
+    # Sanitize email
+    email = user_data['email'].lower().strip()
+    if not '@' in email:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)